site/docs/reference/crds.md

```diff
@@ -3709,6 +3710,116 @@ ObservedState includes information to be reported back to the FybrikApplication
       </tr></tbody>
 </table>
 
+## app.fybrik.io/v1beta2
+
+Resource Types:
+
+- [FybrikStorageAccount](#fybrikstorageaccount)
+
+
+
+
+### FybrikStorageAccount
+<sup><sup>[↩ Parent](#appfybrikiov1beta2 )</sup></sup>
+
+
+
+
+
+
+FybrikStorageAccount defines a storage account used for copying data. It contains connection details of the shared storage and refers to the secret that stores storage credentials.
```

Comment:
FybrikStorageAccount is a storage account Fybrik uses to dynamically allocate space for datasets whose creation or copy it is orchestrates

Reply:
Done